dockerfiles/mlrun-api/Dockerfile

```diff
@@ -63,9 +63,21 @@ ENV MLRUN_HTTPDB__DSN='sqlite:////mlrun/db/mlrun.db?check_same_thread=false'
 ENV MLRUN_HTTPDB__LOGS_PATH=/mlrun/db/logs
 ENV MLRUN_httpdb__dirpath=/mlrun/db
 ENV MLRUN_httpdb__port=8080
+ENV MLRUN_IS_API_SERVER=true
+ENV MLRUN_IGNORE_ENV_FILE=true
+
 COPY . .
 
 RUN python -m pip install .[complete-api]
+
 VOLUME /mlrun/db
 
-CMD ["python",  "-m",  "server.api",  "db"]
+CMD [ \
+    "sh", "-c", \
```

Comment:
Nitpick: I'm not sure `sh` is capable of reaping zombie processes by itself. I'm, however, pretty sure that `bash` is.

However, you might prefer switching to [tini](https://github.com/krallin/tini) so that process signals that the container receives are passed along to `uvicorn`.